src/pages/my-courses/MyCourses.tsx

```diff
@@ -112,7 +151,11 @@ const MyCourses = () => {
         />
       ) : (
         <>
-          <MyCorsesCardsList deleteItem={onDelete} items={coursesItems} />
+          <MyCorsesCardsList
+            deleteItem={onDelete}
+            duplicateItem={(itemId: string) => void handleDuplicate(itemId)}
```

Comment:
can you just pass handleDuplicate to dublicate item:

duplicateItem={handleDuplicate}

Reply:
No, because the promise-returning function was provided to attribute where a void return was expected

Reply:
I guess changing type of this prop in MyCorsesCardsList will fix it, but , maybe it`s not needed